Src/DependencyCollector/Shared.Tests/Implementation/SqlClientDiagnosticSourceListenerTests.cs

```diff
@@ -141,7 +140,7 @@ public void TracksCommandExecuted()
             Assert.IsTrue((bool)dependencyTelemetry.Success);
             Assert.IsTrue(dependencyTelemetry.Duration > TimeSpan.Zero);
             Assert.IsTrue(dependencyTelemetry.Duration < TimeSpan.FromMilliseconds(500));
-            Assert.IsTrue(DateTimeOffset.UtcNow >= dependencyTelemetry.Timestamp);
+            Assert.IsTrue(Math.Abs((start - dependencyTelemetry.Timestamp).TotalMilliseconds) <= 16);
```

Comment:
why the magic number "16" ?

Reply:
This is about datetime.now precision on .net fx
https://stackoverflow.com/questions/2143140/c-sharp-datetime-now-precision

Reply:
Changes in these tests are caused by base sdk which started to set more precise timestamps in https://github.com/microsoft/ApplicationInsights-dotnet/pull/1126